packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -44,21 +47,56 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
   assert('PoolShares' in terms.brands, 'no PoolShares brand');
 
   const statusManager = prepareStatusManager(zone);
-  const feed = prepareTransactionFeed(zone);
   const makeSettler = prepareSettler(zone, { statusManager });
   const { chainHub, vowTools } = tools;
   const makeAdvancer = prepareAdvancer(zone, {
     chainHub,
-    feed,
     log: trace,
     statusManager,
     vowTools,
   });
-  assertAllDefined({ feed, makeAdvancer, makeSettler, statusManager });
+  const makeFeedKit = prepareTransactionFeedKit(zone);
+  assertAllDefined({ makeFeedKit, makeAdvancer, makeSettler, statusManager });
+  const feedKit = makeFeedKit();
+  const advancer = makeAdvancer(
+    // @ts-expect-error FIXME
+    {},
+  );
+  // Connect evidence stream to advancer
+  void observeIteration(subscribeEach(feedKit.public.getEvidenceStream()), {
+    updateState(evidence) {
+      try {
+        advancer.handleTransactionEvent(evidence);
+      } catch (err) {
+        trace('🚨 Error handling transaction event', err);
+      }
+    },
+  });
```

Comment:
Good questions. It may need to be durable but that's for Milestone 2.

If the stream dies, I don't think resubscribing would be safe. It's an unhandled rejection that will need a Core Eval to repair.